actioncable/lib/assets/javascripts/action_cable/connection.coffee

```diff
@@ -75,7 +79,16 @@ class ActionCable.Connection
     error: ->
       @disconnect()
 
+  dispatchEvent: (identifier, eventName, message = null) ->
+    channel = if identifier then JSON.parse(identifier).channel else null
```

Comment:
Channels can have params too. Example from the README:
```coffee
App.cable.subscriptions.create { channel: "ChatChannel", room: "Best Room" }
```
So, properties other than `channel` will need to be included in the event data somehow.